lib/rubygems/package/tar_header.rb

```diff
@@ -161,9 +163,7 @@ def initialize(vals)
     vals[:devmajor] ||= 0
     vals[:devminor] ||= 0
```

Comment:
Instead of the lines like this, just write directly to the instance variable like:

```ruby
@devminor = val[:devminor] || 0
```

Then you avoid strings, loops, method calls and a hash assingment.